src/SignhostAPIClient/Rest/DataObjects/Signer.cs

```diff
@@ -37,8 +37,6 @@ public class Signer
 
 		public bool? SendSignConfirmation { get; set; }
 
-		public int DaysToRemind { get; set; }
```

Comment:
I am afraid we can't just remove this property without releasing a new major version on nuget. So leave this one in here, but mark it with an ObsoleteAttribute and point it to the right property